var/spack/repos/builtin/packages/util-macros/package.py

```diff
@@ -11,13 +11,19 @@ class UtilMacros(AutotoolsPackage, XorgPackage):
     other Xorg modular packages, and is needed to generate new versions
     of their configure scripts with autoconf."""
 
-    homepage = "https://cgit.freedesktop.org/xorg/util/macros/"
-    xorg_mirror_path = "util/util-macros-1.19.1.tar.bz2"
+    homepage = "https://gitlab.freedesktop.org/xorg/util/macros"
+    xorg_mirror_path = "util/util-macros-1.19.1.tar.xz"
 
-    maintainers("robert-mijakovic")
+    # note: url_for_version can only return a single url, no mirrors
+    def url_for_version(self, version):
+        if version < Version("1.20"):
```

Comment:
```suggestion
        if self.spec.satisfies("@:1.19"):
```
If you'd like to cut down on creating another version object. Here's an optional suggestion using `satisfies()`.